aliases/git/nu_alias_git.nu

```diff
@@ -0,0 +1,167 @@
+export alias gpd                   = git push --dry-run
+export alias gpf                   = git push --force-with-lease
+export alias gpf!                  = git push --force
+export alias gpoat                 = (git push origin --all; git push origin --tags)
+export alias gpr                   = git pull --rebase
+export alias gpu                   = git push upstream
+export alias gpv                   = git push -v
+
+export alias gr                    = git remote
+export alias gra                   = git remote add
+export alias grb                   = git rebase
+export alias grba                  = git rebase --abort
+export alias grbc                  = git rebase --continue
+export alias grbi                  = git rebase -i
+export alias grbo                  = git rebase --onto
+export alias grbs                  = git rebase --skip
+export alias grev                  = git revert
+export alias grh                   = git reset
+export alias grhh                  = git reset --hard
+# export alias groh                  = (get_current_branch | git reset origin/$in --hard)
+export alias groh                  = get_current_branch
+export alias grm                   = git rm
+export alias grmc                  = git rm --cached
+export alias grmv                  = git remote rename
+export alias grrm                  = git remote remove
+export alias grs                   = git restore
+export alias grset                 = git remote set-url
+export alias grss                  = git restore --source
+export alias grst                  = git restore --staged
+export alias grt                   = cd (git rev-parse --show-toplevel || echo .)
+export alias gru                   = git reset --
+export alias grup                  = git remote update
+export alias grv                   = git remote -v
+
+export alias gsb                   = git status -sb
+export alias gsd                   = git svn dcommit
+export alias gsh                   = git show
+export alias gsi                   = git submodule init
+export alias gsps                  = git show --pretty=short --show-signature
+export alias gsr                   = git svn rebase
+export alias gss                   = git status -s
+export alias gst                   = git status
+
+export alias gstaa                 = git stash apply
+export alias gstc                  = git stash clear
+export alias gstd                  = git stash drop
+export alias gstl                  = git stash list
+export alias gstp                  = git stash pop
+export alias gsts                  = git stash show --text
+export alias gstu                  = gsta --include-untracked
+export alias gstall                = git stash --all
+export alias gsu                   = git submodule update
+export alias gsw                   = git switch
+export alias gswc                  = git switch -c
+
+export alias gts                   = git tag -s
+export alias gtv                   = (git tag | sort)
```

Comment:
I'm only unsure if this function works since I haven't worked with `git tag` yet but I think it *should*

Reply:
Maybe needs a `lines` to convert the lines from `git tag` to a nushell `sort`able list.

Reply:
good point. I just added it.

Reply:
Thanks! checked that it does alphabetical sorting. For `-n`/`--natural`(nushell)/`--numeric-sort`(GNU coreutils) sorting, the nushell pipeline aliases are a bit different as they don't allow you to append a flag to the last pipeline step (I think this could work in bash as there much more stuff is just text)